controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -90,11 +90,25 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	case len(instances.Items) == 1:
 		instance = &instances.Items[0]
 	case len(instances.Items) > 1:
-		message := fmt.Sprintf("only one instance of DSCInitialization object is allowed. Update existing instance name %s", req.Name)
-		_, _ = r.updateStatus(ctx, &instances.Items[0], func(saved *dsciv1.DSCInitialization) {
-			status.SetErrorCondition(&saved.Status.Conditions, status.DuplicateDSCInitialization, message)
-			saved.Status.Phase = status.PhaseError
-		})
+		// find out the one by created timestamp and use it as the default one
+		earliestDSCI := &instances.Items[0]
+		for _, instance := range instances.Items {
+			currentDSCI := instance
+			if currentDSCI.CreationTimestamp.Before(&earliestDSCI.CreationTimestamp) || currentDSCI.CreationTimestamp.Equal(&earliestDSCI.CreationTimestamp) {
```

Comment:
```suggestion
			if currentDSCI.CreationTimestamp.Before(&earliestDSCI.CreationTimestamp) {
```

I don't think the latter will ever happen, so we can simplify.

Reply:
it actually was, that's why i added it there.  
in the example of {"new", "old"}: when "new" is comparing with "new"